test/src/services/update_acsp/updateYourDetailsService.test.ts

```diff
@@ -184,4 +184,9 @@ describe("CheckYourAnswersService", () => {
             serviceAddress: ""
         });
     });
+
+    it("should return correct business address removing ACSP", () => {
```

Comment:
You can add 3 more tests where the name contains ACSP but in lower case, one with ACSP in the name at all and one with ACSP/acsp in the middle of the name